### PR TITLE
Ability to resize scrollbar and improvements

### PR DESCRIPTION
**Pull request by edunad**
First of all, amazing package! I've made a few improvements :
- Updated packages
- Ability to resize scrollbar
- Replace highlight timer with css animation

### CODE REVIEW

**Comment by RevolvingDCON** [comment]
This is awesome!

**Comment by edunad** [comment]
@kmck toughs?

File: src/utils/index.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ export function getScrollbarSize(parentNode = document.body) {
   outer.setAttribute('data-measure-scrollbar', 'true');
   outer.style.visibility = 'hidden';
   outer.style.overflow = 'scroll';
-  outer.style.msOverflowStyle = 'scrollbar';
```
Comment (kmck) [inline src/utils/index.ts:15]: why remove this?

Comment (edunad) [reply src/utils/index.ts:15]: It was giving out some compiling issues, not sure if it's my setup

**Comment by edunad** [comment]
Updated :D

**Pull request merged**